src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Search/NewAzureOperationalInsightsComputerGroupCommand.cs

```diff
@@ -73,11 +73,6 @@ protected override void ProcessRecord()
                 Tags = new List<Tag>() { new Tag() { Name = "Group", Value = "Computer" } }
             };
 
-            if (!SearchCommandHelper.IsListOfComputers(this.Query))
```

Comment:
I don't see any test coverage for the new query format? Also is there any tests that make sure the old query will still work. Also can a user combine the new format with the old? If yes would be great to add that in the test coverage.